README.md

```diff
@@ -71,6 +71,24 @@ You will be prompted to supply a username, email, and password.
 
 Having completed that process, you will be able to access the admin interface at `/admin` using your new superuser account.
 
+### Create fake finding data
+
+Once you can access the admin interface for the project, you'll be able to create fake finding data locally.
+
+Start by adding and saving a new Finding record: http://127.0.0.1:8000/admin/resolve_findings/finding/add/.
+
+### See the project locally
+
+Once your new finding record is saved, visit http://127.0.0.1:8000/finding/1/ to see the Finding Resolution Page.
+
+Here is what that page looks like as of June 2019:
+
+![screenshot of Finding Resolution page](https://user-images.githubusercontent.com/3209501/59533745-04f50480-8eb2-11e9-9dab-1ecdd3498b43.png)
```

Comment:
It'd be ideal to have the agency names here be imaginary... @bpdesigns, have you already updated the mockups to include that? If not, it doesn't need to block merging this, but let's make a follow-up ticket.

Reply:
Scratch that, @bpdesigns — these are screenshots of @alexsoble's current implementation! :) @alexsoble, would you mind updating your local copy (and the screenshots) accordingly?

Reply:
Updating the title to `Single Audit Finding Resolution` to avoid an unfortunate acronym would also be 👍

Reply:
@fureigh Updated title in #125.

Reply:
@fureigh Updated README screenshots, you can take a look at the newest here: 

https://github.com/18F/federal-grant-reporting/blob/871539db93a0740d519b689aa57b147bf9cdcb77/README.md